docs/core/introduction.md

```diff
@@ -6,404 +6,52 @@ ms.date: 02/24/2022
 ms.custom: "updateeachrelease"
 recommendations: false
 ---
-# What is .NET? Introduction and overview
+# What is .NET?
 
-.NET is a free, open-source development platform for building many kinds of apps, such as:
+NET is a free, cross-platform, open source developer platform for building many different types of [applications](tutorials/with-visual-studio-code.md). .NET is built on a [high-performance runtime](https://devblogs.microsoft.com/dotnet/category/performance/) that is used in production by many [high-scale apps](https://devblogs.microsoft.com/dotnet/category/app/). It is built with the community on [GitHub](https://github.com/dotnet/core).
 
-* [Web apps, web APIs, and microservices](/aspnet/core/introduction-to-aspnet-core#recommended-learning-path)
-* [Serverless functions in the cloud](/azure/azure-functions/functions-create-first-function-vs-code?pivots=programming-language-csharp)
-* [Cloud native apps](../architecture/cloud-native/index.md)
-* [Mobile apps](https://dotnet.microsoft.com/learn/xamarin/hello-world-tutorial/intro)
-* Desktop apps
-  * [Windows WPF](/dotnet/desktop/wpf/)
-  * [Windows Forms](/dotnet/desktop/winforms/)
-  * [Universal Windows Platform (UWP)](/windows/uwp/get-started/create-a-hello-world-app-xaml-universal)
-* [Games](https://dotnet.microsoft.com/apps/games)
-* [Internet of Things (IoT)](../iot/index.yml)
-* [Machine learning](../machine-learning/index.yml)
-* [Console apps](tutorials/with-visual-studio-code.md)
-* [Windows services](/aspnet/core/host-and-deploy/windows-service)
+You can use .NET for client apps, for [desktop](/dotnet/desktop/), [mobile](/xamarin/), [browser](aspnet/core/blazor), and [games](https://dotnet.microsoft.com/apps/games).
```

Comment:
should we say web instead browser? Also, with Blazor now coming into play with hybrid apps, should we link to aspnet/core instead?

Reply:
I just checked. They are called "client web" apps. We're already talking about client apps, so I guess "web" it is.

https://dotnet.microsoft.com/en-us/apps/aspnet/web-apps/blazor

```suggestion
You can use .NET for client apps, for [desktop](/dotnet/desktop/), [mobile](/xamarin/), [web](aspnet/core/blazor), and [games](https://dotnet.microsoft.com/apps/games).
```